app/src/components/EditEvent/EditEvent/EditEvent.tsx

```diff
@@ -265,13 +280,40 @@ export const EditEvent = ({ eventResult: event, updateEvent }: IProps) => {
         }
       />
       <Checkbox
-        label="Eksternt arrangement"
-        onChange={(isExternal) => updateEvent({ ...event, isExternal })}
-        isChecked={event.isExternal}
         onDarkBackground
+        label="Arrangementet er eksternt"
+        isChecked={event.isExternal}
+        onChange={(isExternal) => updateEvent({ ...event, isExternal })}
       />
+
+      {isCreateView && (
+        <>
+          <Checkbox
+            onDarkBackground
+            label={'Legg til et kortnavn (pen URL)'}
+            isChecked={hasShortname}
+            onChange={setHasShortname}
+          />
+          {hasShortname && (
+            <div className={style.flex}>
+              <div style={{ maxWidth: 'min-content' }}>
+                {document.location.origin}/
+              </div>
+              <div>
+                <ValidatedTextInput
+                  label={''}
+                  value={event.shortname || ''}
+                  onChange={(shortname) => updateEvent({ ...event, shortname })}
+                  validation={parseShortname}
+                />
+              </div>
+            </div>
+          )}
+        </>
+      )}
+
```

Comment:
Kan man kun sette kortnavnet ved opprettelse og ikke endre senere? 
Hvis ikke det er veldig vanskelig å gjøre burde det kanskje være mulig. F.eks. hvis noen oppretter et arrangement og i etterkant kommer på at et pent navn hadde vært lurt.

Reply:
Ja det er sånn det funker 😂 Problemet er at da må vi legge inn kode for å lese ut current kortnavn til arrangementet (og opprette tilhørende endepunkt backend), for så å sende med nytt kortnavn (og da må backenden endres til å ta hensyn til dette og så slette eksisterande kortnavn dersom det er undefined i writemodellen). Så var bare masse knot eg ville unngå for å prøve å få inn funksjonaliteten litt sånn MVP. Men det blir jo sikkert eit poeng.